Add Aqua Cloud Native Security Platform, Datadog Agent and Self-hosted Runner files (#113)
odidev/ecosystem-dashboard-for-arm: content/commercial_packages/Datadog_Agent.md

---
name: Datadog Agent
vendor: Datadog
category: Monitoring/Observability

> description: The Datadog Agent is software that runs on your hosts. It collects events and metrics from hosts and sends them to Datadog, where you can analyze your monitoring and performance data.

Author: ```suggestion
description: The Datadog Agent is software that runs on the hosts. It collects events and metrics from hosts and sends them to Datadog, where the monitoring and performance data can be analyzed.
```
